### PR TITLE
Add a call to metrics for incrementing a module+model in buffers.

### PR DESCRIPTION
**Pull request by JTCunning**
lmk if this is a bad idea. I've fallen off the sentry-code wagon for some time now.

### CODE REVIEW

File: src/sentry/buffer/redis.py [hunk 1]
```diff
@@ -140,6 +140,11 @@ def incr(self, model, columns, filters, extra=None):
         pipe.zadd(pending_key, time(), key)
         pipe.execute()
 
+        metrics.incr('buffer.incr', tags={
+            'module': model.__module__,
+            'model': model.__name__,
+        })
```
Comment (mattrobenolt) [inline src/sentry/buffer/redis.py]: Pass `skip_internal=True`

Comment (JTCunning) [reply src/sentry/buffer/redis.py]: GOOD IDEA

File: src/sentry/buffer/redis.py [hunk 2]
```diff
@@ -140,6 +140,11 @@ def incr(self, model, columns, filters, extra=None):
         pipe.zadd(pending_key, time(), key)
         pipe.execute()
 
+        metrics.incr('buffer.incr', skip_internal=True, tags={
```
Comment (mattrobenolt) [inline src/sentry/buffer/redis.py]: I'd send just:

```python
metrics.incr('buffer.incr', instance=':'.join((model.__module__, model.__name__)), skip_internal=True)
```

Then you'll get an `instance` type as `sentry.models.user:User` which is a more, full path to the model. Unless there's a reason you want them actually split into separate tags, then ignore me.

**Review by mattrobenolt — APPROVED** [review]

File: src/sentry/buffer/redis.py (hunk 2 above)
Comment (JTCunning) [reply src/sentry/buffer/redis.py]: I'm not too big on our `instance` tag. I think that having both makes it easier to read in graphs, and in the eventuality that we're only coming out of one module, separating by model will make this more digestible.

**Pull request merged**